myapp/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,43 @@
+class UsersController < ApplicationController
+  skip_before_action :logged_in_user, only: %i[new create]
+  before_action :editing_authority, only: %i[edit update]
+
+  def new
+    redirect_to root_path and return unless current_user.nil?
+    @user = User.new
+  end
+
+  def edit
+    @user = User.find(params[:id])
```

Comment:
```suggestion
    @user = current_user
```
にするとparams受け取らなくてよくなるので
セキュリティ的にも良さそうですし、editing_authorityで権限チェックしなくてよくなりそうです

Reply:
（もちろん、adminユーザーが他のユーザーの情報を修正するとかだったらparamsでidを受け取らないとですが）

Reply:
こちら次のSTEPで管理者が修正できるようにしているため、
一旦上記の状態を維持させていただければと思います:pensive:

Reply:
あ、次のステップで管理機能実装ですか。承知しました